text/0821-public-types.md

```diff
@@ -283,7 +284,7 @@ class _Transition<T = unknown>
   implements Pick<Promise<T>, 'then' | 'catch' | 'finally'> {
 
   data: Record<string, unknown>;
-  readonly from: RouteInfoWithAttributes<T> | null;
+  readonly from: RouteInfoWithAttributes<T> |;
```

Comment:
We should probably revert these changes before merging?
These changes were caused by a bug in `rfcs-tooling`, but has now been resolved.

Reply:
Yeeeep, thanks for catching it!